src/server.c

```diff
@@ -5490,10 +5490,10 @@ sds genValkeyInfoStringCommandStats(sds info, hashtable *commands) {
         char *tmpsafe;
         if (c->calls || c->failed_calls || c->rejected_calls) {
             info = sdscatprintf(info,
-                                "cmdstat_%s:calls=%lld,usec=%lld,usec_per_call=%.2f"
+                                "cmdstat_%s:calls=%lld,usec=%lld,usec_per_call=%lld"
```

Comment:
This will change the format slightly from `1.00` to `1`.

I think it's right to remove the sub-millisecond part. We only measure the execution time in microseconds anyway, so this is already not exact on sub-millisecond level.

We should probably label it as a potentially breaking change though, just in some case some tool is parsing this and expect the period to be there.

Reply:
I think sub-microsecond latency is useful - why not modify `getMonotonicUs` to return nanoseconds instead (getMonotonicNs)? both  __rdtsc/__cntvct  can provide sub-microsecond precision...